engines/ospray/utils.h

```diff
@@ -63,4 +63,30 @@ std::vector<std::array<float, S>> convertVectorToFloat(
     }
     return output;
 }
-}
+
+namespace osphelper
+{
```

Comment:
I think it's better to make these functions inline. It wouldn't be a premature optimization, it's about avoiding unreasonable penalizations.

Reply:
I prefer to keep them in the cpp file to keep the header clean and since it is not performance critical I think that trumps the performance loss.